src/frontend/web_application/src/index.jsx

```diff
@@ -2,11 +2,19 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { ConnectedRouter } from 'react-router-redux';
 import { AppContainer } from 'react-hot-loader';
+import PiwikReactRouter from 'piwik-react-router';
+import { v1 as uuidv1 } from 'uuid';
 import App from './App';
 import configureStore from './store/configure-store';
 import getRouterHistory from './services/router-history';
 import { getLocale } from './services/i18n';
 
+const piwik = PiwikReactRouter({
+  url: 'https://piwik.caliopen.org/analytics',
+  siteId: 6,
+  userId: uuidv1(), //userId can be username, email adress or uuid
```

Comment:
I don't know if this will work well. May be it's enough : the userId is generated only one time by "browser session".

But we will miss the association with real user. May be it can be done only if we can update the piwik's configuration on the fly (when user is authenticated, I mean).